src/app/core/entity/schema-datatypes/datatype-month.ts

```diff
@@ -28,6 +28,9 @@ import { EntitySchemaDatatype } from "../schema/entity-schema-datatype";
  */
 export const monthEntitySchemaDatatype: EntitySchemaDatatype = {
   name: "month",
```

Comment:
shall we prefix all the related, e.g. "date-month", "date", "date-time" etc.? Or stick to independent but concise names like "month"?

Reply:
I don't see a big benefit of using `date-month` instead of just `month`